main/src/cgeo/geocaching/ui/recyclerview/ManagedListAdapter.java

```diff
@@ -168,8 +168,9 @@ public void clearView(@NonNull final RecyclerView recyclerView, @NonNull final R
                 //if necessary, pass change event to views with position change
                 if (currentDragItemStart >= 0) {
                     if (notifyOnPositionChange && currentDragItemStart != viewHolder.getAdapterPosition()) {
-                        notifyItemChanged(currentDragItemStart);
-                        notifyItemChanged(viewHolder.getAdapterPosition());
+                        for (int item = Math.min(currentDragItemStart, viewHolder.getAdapterPosition()); item <= Math.max(currentDragItemStart, viewHolder.getAdapterPosition()); item++) {
```

Comment:
Good catch, that was a bug indeed! But you may also use notifyItemRangeChange here so you don't have to use a loop

Reply:
> But you may also use notifyItemRangeChange here so you don't have to use a loop

Ok. Thanks! I didn't know that...